rrule/src/tests/edge_cases.rs

```diff
@@ -0,0 +1,38 @@
+use crate::RRuleSet;
+
+#[test]
+fn non_ascii_string() {
```

Comment:
Do you think thes tests in this file could be added to `rrule/tests/regressions.rs`?